src/vivarium/framework/engine.py

```diff
@@ -184,13 +186,15 @@ def __init__(self, configuration, plugin_manager):
         self.value = plugin_manager.get_plugin_interface('value')                   # type: ValuesInterface
         self.event = plugin_manager.get_plugin_interface('event')                   # type: EventInterface
         self.population = plugin_manager.get_plugin_interface('population')         # type: PopulationInterface
-        self.resources = plugin_manager.get_plugin_interface('resource')             # type: ResourceInterface
+        self.resources = plugin_manager.get_plugin_interface('resource')            # type: ResourceInterface
+        self.results = plugin_manager.get_plugin_interface('results')               # type: ResultsInterface
```

Comment:
Need to add docs for this to match the api docs in my PR

Reply:
I'll put it on my list for when the system is implemented.